### PR TITLE
Add detail to some descriptions

### PR DESCRIPTION
**Pull request by evazorro**
Experimenting with longer descriptions on model inputs. Could be useful to add some of these into the regular [SD3 page](https://replicate.com/stability-ai/stable-diffusion-3)?

### CODE REVIEW

File: predict.py [hunk 1]
```diff
@@ -95,38 +95,38 @@ def predict(
...
         ),
         scheduler: str = Input(
-            description="The scheduler to use for the diffusion model",
+            description="The scheduler to use for the diffusion model (used to manage noise; do not use karras)",
             choices=SCHEDULERS,
             default="sgm_uniform",
         ),
         shift: float = Input(
-            description="The timestep scheduling shift. Try values 6.0 and 2.0 to experiment with effects.",
+            description="The timestep scheduling shift; higher shift values are better at managing noise in higher resolutions. Try values 6.0 and 2.0 to experiment with effects.",
```
Comment (zeke) [inline predict.py]: > Try values 6.0 and 2.0

Is this intentional? They seem like to two random values.

**Review by zeke — APPROVED** [review]
Nice! One little comment, but this is definitely an improvement. 👍🏼

File: predict.py [hunk 2]
```diff
@@ -95,38 +95,38 @@ def predict(
         model: str = Input(
             choices=SD3_MODELS,
             default="sd3_medium_incl_clips_t5xxlfp16.safetensors",
-            description="Pick whether to use T5-XXL in fp16, fp8 or not at all",
+            description="Pick whether to use T5-XXL in fp16, fp8 or not at all. We recommend fp8 for simplicity and best results. fp16 improves image quality at the cost of higher memory usage.",
```
Comment (fofr) [inline predict.py]:
```suggestion
            description="Pick whether to use T5-XXL in fp16, fp8 or not at all. We recommend fp16 for this model as it has the best image quality. When running locally we recommend fp8 for lower memory usage. We've included all versions here for exploration.",
```

File: predict.py [hunk 3]
```diff
@@ -95,38 +95,38 @@ def predict(
...
             default=1024,
         ),
         height: int = Input(
-            description="The height of the image",
+            description="The height of the image (best output at ~1 megapixel. Resolution must be divisible by 64)",
             default=1024,
         ),
         steps: int = Input(
-            description="The number of steps to run the diffusion model for",
+            description="The number of steps to run the diffusion model for (more steps = better image but slower generation. Best results for this model around 26 to 36 steps.)",
```
Comment (fofr) [inline predict.py]:
```suggestion
            description="The number of steps to run the diffusion model for (more steps = better image but slower generation. Best results for this model are around 26 to 36 steps.)",
```

File: predict.py (hunk 1 above)
Comment (fofr) [reply predict.py]: These were the high scoring values in the human evaluation of model shift in the SD3 paper.

File: predict.py [hunk 4]
```diff
@@ -95,38 +95,38 @@ def predict(
...
             le=20,
             ge=0,
             default=3.0,
         ),
         guidance_scale: float = Input(
-            description="The guidance scale tells the model how similar the output should be to the prompt.",
+            description="The guidance scale tells the model how similar the output should be to the prompt. (Recommend between 3.5 and 4.5; if images look 'burnt,' lower the value.)",
             le=20,
             ge=0,
             default=4.5,
```
Comment (fofr) [inline predict.py]:
```suggestion
            default=3.5,
```

**Review by fofr — APPROVED** [review]
These look great, I've tweaked a couple and I will merge and deploy.

File: predict.py [hunk 5]
```diff
@@ -95,38 +95,38 @@ def predict(
...
         ),
         scheduler: str = Input(
-            description="The scheduler to use for the diffusion model",
+            description="The scheduler to use for the diffusion model (used to manage noise; do not use karras)",
             choices=SCHEDULERS,
             default="sgm_uniform",
         ),
         shift: float = Input(
-            description="The timestep scheduling shift. Try values 6.0 and 2.0 to experiment with effects.",
+            description="The timestep scheduling shift; higher shift values are better at managing noise in higher resolutions. Try values 6.0 and 2.0 to experiment with effects.",
```
Comment (fofr) [inline predict.py]:
```suggestion
            description="The timestep scheduling shift; shift values higher than 1.0 are better at managing noise in higher resolutions. Try values 6.0 and 2.0 to experiment with effects.",
```

**Pull request merged**